pyuvdata/tests/test_miriad.py

```diff
@@ -923,13 +925,19 @@ def test_readMiriadwriteMiraid_check_time_format():
 
```

Comment:
I've tried to test this with my internet connection off, and unfortunately the tests fail for me... On the line where `delta_lst` is calculated, I get an `IERSRangeError`. I've tried to explicitly set `iers.conf.auto_max_age = None` in the course of the test, but that still raises the error. Does this happen for you locally?

Reply:
No, it doesn't. What version of astropy do you have?

Reply:
I'm also on astropy v2.0.11. The tests pass for me if I've previously downloaded the IERS table and then have no internet connection, or if I have an active internet connection. (Interestingly, in both cases I get a warning about the time being outside of the IERS range...) But if I clear the download cache and try without internet, I still get the `IERSRangeError`.

I'm inclined to just approve this anyway though, because the tests on travis do not raise the warning I'm seeing locally, and so my local install might be misbehaving in some way. It might be nice to see if tests for a third user pass or fail, but honestly I'm also okay approving now.

Reply:
Did you use an existing environment or did you make a new one?